kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -114,3 +114,73 @@ namespace          domain   cluster    min  max  goal  current  ready
+```
+
+Use the following command to roll the WebLogic Cluster Servers with a specific `restartVersion`:
+```
+$ rollCluster.sh -r v2 -c cluster-1 -d domain1 -n weblogic-domain-1
+[2021-03-24T13:46:16.833 UTC][INFO] Patching restartVersion for cluster 'cluster-1' to 'v2'.
+domain.weblogic.oracle/domain1 patched
+[2021-03-24T13:46:16.975 UTC][INFO] Successfully patched replicas for cluster 'cluster-1'!
+```
+
+### Scripts to restart a WebLogic Server in a domain
+The `restartServer.sh` script can be used to restart a WebLogic Server in a domain. This script restarts the Server by deleting the Server Pod for the WebLogic Server instance.
+```
+$ restartServer.sh -s managed-server1 -d domain1 -n weblogic-domain-1
+[2021-03-24T22:20:22.498 UTC][INFO] Initiating restart of 'managed-server1' by deleting server pod 'domain1-managed-server1'.
+[2021-03-24T22:20:37.614 UTC][INFO] Server restart succeeded !
+```
+
+### Scripts to explicitly initiate introspection of a WebLogic domain
+
+The `introspectDomain.sh` script can be used to rerun a WebLogic domain's introspect job by explicitly initiating the introspection. This script updates the value of `spec.introspectVersion` attribute of the domain resource. The resulting behavior depends on your domain home source type and other factors, see [Initiating introspection](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/introspection/#initiating-introspection) for details. You can provide the new value of `introspectVersion` as a parameter to the script or the script will automatically generate a new value to trigger the introspection. See the script `usage` information by using the `-h` option.
+
+Use the following command to rerun a domain's introspect job with the `introspectVersion` value generated by the script.
+```
+$ introspectDomain.sh -d domain1 -n weblogic-domain-1
+[2021-03-24T21:37:55.989 UTC][INFO] Patching introspectVersion for domain 'domain1' to '1'.
+domain.weblogic.oracle/domain1 patched
+[2021-03-24T21:37:56.110 UTC][INFO] Successfully patched introspectVersion for domain 'domain1'!
+```
+
+Use the following command to rerun a domain's introspect job with a specific `introspectVersion` value.
+```
+$ introspectDomain.sh -i v1 -d domain1 -n weblogic-domain-1
+[2021-03-24T21:38:34.369 UTC][INFO] Patching introspectVersion for domain 'domain1' to 'v1'.
+domain.weblogic.oracle/domain1 patched
+[2021-03-24T21:38:34.488 UTC][INFO] Successfully patched introspectVersion for domain 'domain1'!
+```
+
+### Watching the Pods after executing life cycle scripts
+
+After executing the various life cycle scripts described above for a domain or a cluster or a Server, you can manually run the `kubectl -n MYNS get pods --watch=true --show-labels` command to watch the effects of running the script and monitor the status and labels of various Pods. You will need to run 'Ctrl-C' in the end to stop watching the Pods and exit.
```

Comment:
the various life cycle scripts -> the lifecycle scripts (life cycle (n), lifecycle (adj) 
to watch the effects -> to watch the effect
in the end to stop -> to stop